tasks/tiler/xyz.py

```diff
@@ -83,7 +83,7 @@ def _create_table(self, session, config):
                        bounds Numeric[],
                        envelope Geometry,
                        ext Geometry,
-                       CONSTRAINT {table}_pk PRIMARY KEY (x,y,z)
+                       CONSTRAINT {table}_pk PRIMARY KEY (z,x,y)
```

Comment:
Does this mean we need to rerun all existing XYZ generation and not only zooms 8 and 9?

Reply:
Not at all. This is just a convenience. For the application, which always sends x, y and z, this means nothing. But, for some operations, mostly manual ones but from time to time also in the ETL (such as today's "let's change some zoom levels"), having z as the first element of the index makes some them faster.

Reply:
Then I guess we can just merge this and run in production to get the new XYZ dump (or just checkout this branch in production as acceptance+deploy).